### ISSUE TITLE
[A11y]: Design System [Component] [Demo Content]: Magnification - CONTINUE button is partially cut off when viewing at 300% zoom

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#45874]
### Point of contact

Angela Fowler

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

...
## User Story or Problem Statement

As a user who magnifies content on a page, I expect all content to be visible without horizontal scroll.

## Details

Currently, the CONTINUE button is partially cut off from view at 300% magnification (1280px screen size).
## Acceptance Criteria

- [ ] CONTINUE button on all form steps is completely visible and not cut off

## Environment

* macOS Monterey, `<Version 12.4>`

* Chrome, `<Version 102.0.5005.61>`

* VoiceOver

* Staging
* 

### Reproduction steps
1. Enter `https://staging.va.gov/burial-poc-v6/` in browser address bar

2. Navigate through all of the form steps with screen set to 1280px and zoom to 300%

3. Verify that the CONTINUE button is partially cut off

### Proposed solution or next steps

...

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [x] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [x] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by jeana-adhoc** [comment]
I was looking for other button pair issue tickets, and found this one, and I'm not sure it's happening any more. I Tested this form, and the buttons now stack one on top of the other in the form, and all text is legiible and not cut off even to 400% zoom.

**Comment by caw310** [comment]
@rsmithadhoc please confirm that this is no longer and issue. If it isn't please close. Thanks!

**Comment by rsmithadhoc** [comment]
Confirmed this is not an issue any more, closing this out.
